src/Client/ClickHouseClient.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
```

Comment:
Does it worth doing

1. Create a subfolder called `ClickHouse` under `Client` to separate the source code at file system folder layer. I feel like we may have more things to stuff here
2. Create a namespace called `ch` or `CH`, so DB::ch or DB::CH::Client instead of ClickHouseClient etc

Reply:
I name the namespace `ClickHouse`, since `ch` or `CH` is ambiguous.